graphrag/query/cli.py

```diff
@@ -179,3 +175,184 @@ async def run_streaming_search():
+            )
+            dataframe_dict["community_reports"] = pd.read_parquet(
+                path=get_abfs_path("create_final_community_reports.parquet"),
+                storage_options=storage_options,
+            )
+            dataframe_dict["text_units"] = pd.read_parquet(
+                path=get_abfs_path("create_final_text_units.parquet"),
+                storage_options=storage_options,
+            )
+            dataframe_dict["relationships"] = pd.read_parquet(
+                path=get_abfs_path("create_final_relationships.parquet"),
+                storage_options=storage_options,
+            )
+
+        case StorageType.file:
+            dataframe_dict["nodes"] = pd.read_parquet(
+                data_path / "create_final_nodes.parquet"
+            )
+            dataframe_dict["entities"] = pd.read_parquet(
+                data_path / "create_final_entities.parquet"
+            )
+            dataframe_dict["community_reports"] = pd.read_parquet(
+                data_path / "create_final_community_reports.parquet"
+            )
+            dataframe_dict["text_units"] = pd.read_parquet(
+                data_path / "create_final_text_units.parquet"
+            )
+            dataframe_dict["relationships"] = pd.read_parquet(
+                data_path / "create_final_relationships.parquet"
+            )
+
+            final_covariates_path = data_path / "create_final_covariates.parquet"
+            dataframe_dict["covariates"] = (
+                pd.read_parquet(final_covariates_path)
+                if final_covariates_path.exists()
+                else None
+            )
+        case _:
+            dataframe_dict["nodes"] = pd.read_parquet(
+                data_path / "create_final_nodes.parquet"
+            )
+            dataframe_dict["entities"] = pd.read_parquet(
+                data_path / "create_final_entities.parquet"
+            )
+            dataframe_dict["community_reports"] = pd.read_parquet(
+                data_path / "create_final_community_reports.parquet"
+            )
+            dataframe_dict["text_units"] = pd.read_parquet(
+                data_path / "create_final_text_units.parquet"
+            )
+            dataframe_dict["relationships"] = pd.read_parquet(
+                data_path / "create_final_relationships.parquet"
+            )
+
+            final_covariates_path = data_path / "create_final_covariates.parquet"
+            dataframe_dict["covariates"] = (
+                pd.read_parquet(final_covariates_path)
+                if final_covariates_path.exists()
+                else None
+            )
+    return dataframe_dict
+
+
+def _create_graphrag_config(
+    root: str | None,
+    config_filepath: str | None,
+) -> GraphRagConfig:
+    """Create a GraphRag configuration."""
+    return _read_config_parameters(root or "./", config_filepath)
+
+
+def _read_config_parameters(root: str, config: str | None):
```

Comment:
Don't we have something similar in other locations to read a config from a path?

Pretty sure it is on graphrag.config.load_config -> load_config()